packages/node_modules/samples/browser-single-party-call/test/wdio/spec/dial-and-reject.js

```diff
@@ -14,6 +14,10 @@ describe(`samples/browser-single-party-call`, () => {
         mccoy = users[1];
       }));
 
+    // the inner-most function is executed in the window environment
+    // eslint-disable-next-line no-undef
+    before(() => browser.execute(() => window.reload));
```

Comment:
Why not just `browser.reload()`?

Reply:
interesting...thought i looked for that. I'll make the change.